### ISSUE TITLE
I want to testing swiftMonkey on a real Device.

### ISSUE THREAD

**Issue by Mikaly** [zalando/SwiftMonkey#55]
I have do like this,but it does not work! You see.

======================================

Hi @monkeytest15! It's great to hear that you like the tool :) I am also happy to inform you that testing on a real device should be already possible! Just grab the latest changes from master and you should be all set.If you are using CocoaPods to integrate the library just type in your Podfile:pod 'SwiftMonkey', :git => 'https://github.com/zalando/SwiftMonkey.git'The latest master version of SwiftMonkey is not yet released to CocoaPods repository. We should do this soon and then you won't need to explicitly specify the repository URL.
--

**Comment by wojciechczerski** [comment]
Hello @Mikaly ! Could you please provide some more details on what is not working for you?

**Comment by klaudiaem** [comment]
UP!
In my case Xcode complain about 

`ld: '/Applications/Xcode-9.2.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/Library/Frameworks/XCTest.framework/XCTest' does not contain bitcode. You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target. file '/Applications/Xcode-9.2.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/Library/Frameworks/XCTest.framework/XCTest' for architecture armv7
clang: error: linker command failed with exit code 1 (use -v to see invocation)`

as temporary workaround I disable bit code for all but I need that to my project 
`post_install do |installer|
 installer.pods_project.targets.each do |target|
   target.build_configurations.each do |config|
     config.build_settings['ENABLE_BITCODE'] = 'NO'
   end
 end
end`

¯\_(ツ)_/¯

**Comment by wojciechczerski** [comment]
Hey @klaudiaem ! As a workaround I could suggest changing your `post_install` hook to:
```ruby
post_install do |installer|
  installer.pods_project.targets.each do |target|
    if target.name == 'SwiftMonkey'
      target.build_configurations.each do |config|
        config.build_settings['ENABLE_BITCODE'] = 'NO'
      end
    end
  end
end
```
This way only SwiftMonkey will have Bitcode disabled.

Regarding the real solution, I think what's missing is the following entry in the `Podspec`:
```
s.pod_target_xcconfig = { 'ENABLE_BITCODE' => 'NO' }
```

I will create a branch with this change and will test if this indeed solves the problem. If so, I'll make a new release and will notify you of course :)

**Comment by wojciechczerski** [comment]
I have created a branch with the change I mentioned and it seems to do the job! Could you please check if that would work for you as well @klaudiaem ? Just integrate SwiftMonkey in your project using:
```
pod 'SwiftMonkey', :git => 'https://github.com/zalando/SwiftMonkey.git', :branch => 'disable-bitcode'
```
Also you can remove the `pod_install` hook.

**Comment by wojciechczerski** [comment]
Hi @klaudiaem ! I just released SwiftMonkey v2.1.0 which contains the mentioned change to disable Bitcode in the `podspec` file.
Does that solve your issue with testing on a real device?
